_config/security.yml

```diff
@@ -1,4 +1,32 @@
-SilverStripe\Security\MemberLoginForm:
+---
+Name: coresecurity
+---
+SilverStripe\Security\MemberAuthenticator\LoginForm:
   required_fields:
     - Email
-    - Password
\ No newline at end of file
+    - Password
+
+SilverStripe\Core\Injector\Injector:
+  SilverStripe\Control\RequestProcessor:
+    properties:
+      filters:
+        - %$SilverStripe\Security\AuthenticationRequestFilter
+  SilverStripe\Security\MemberAuthenticator\SessionAuthenticationHandler:
+    properties:
+      SessionVariable: loggedInAs
+  SilverStripe\Security\MemberAuthenticator\CookieAuthenticationHandler:
+    properties:
+      TokenCookieName: alc_enc
+      DeviceCookieName: alc_device
+      CascadeLogInTo: %$SilverStripe\Security\MemberAuthenticator\SessionAuthenticationHandler
+  SilverStripe\Security\IdentityStore:
+    class: SilverStripe\Security\AuthenticationRequestFilter
+  SilverStripe\Security\Security:
+    properties:
+      authenticators:
+        default: %$SilverStripe\Security\MemberAuthenticator\MemberAuthenticator
+        cms: %$SilverStripe\Security\MemberAuthenticator\CMSMemberAuthenticator
+SilverStripe\Security\AuthenticationRequestFilter:
```

Comment:
Moving this one up into the Injector chain has proven to be quite unstable. Injection not happening at somewhat random (Somewhere in Injector, the properties weren't called).

That's why the AuthenticationRequestFilter supports both, injection as well as the fallback array. I've opted for the fallback method, to give the option to have a non-API-breaking change in the future.